packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -426,6 +426,7 @@ export default mixins(
 					saveAs(blob, workflowName + '.json');
 				} else if (key === 'workflow-save') {
 					this.saveCurrentWorkflow(undefined);
+					this.$store.dispatch('settings/fetchPromptsData');
```

Comment:
what should happen if save fails? should we still show a prompt?

Reply:
Nope. I added in the callback to check whether the WF were successfully saved or not.